man/kernel-command-line.xml

```diff
@@ -90,6 +90,7 @@
         <term><varname>systemd.default_standard_output=</varname></term>
         <term><varname>systemd.default_standard_error=</varname></term>
         <term><varname>systemd.setenv=</varname></term>
+        <term><varname>systemd.machine_id=</varname></term>
```

Comment:
This should probably be `machine-id`, with a dash rather than an underscore. @poettering, any hard opinion on this?